.github/workflows/R-check.yaml

```diff
@@ -51,7 +57,10 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::rcmdcheck
+          extra-packages: |
+            any::rcmdcheck
+            lme4=?ignore-before-r=4.0.0
+            emmeans=?ignore-before-r=3.6.0
```

Comment:
What do these do?

Reply:
It skips downloading these before a certain R version because they are not available for those R versions.